### PR TITLE
Added double import support

### PR DESCRIPTION
**Pull request by clementpadovani**
Adds fixes for #9

### CODE REVIEW

File: ImportExtension/AddImportOperation.swift [hunk 1]
```diff
@@ -41,6 +47,30 @@ class AddImportOperation {
...
+                if (potentialRemovalInstructionsString.trimmingCharacters(in: CharacterSet.newlines) == AddImportOperationConstants.doubleImportRemoveInstructionsString)
+                {
+                    self.buffer.lines.removeObjects(in: NSMakeRange(selectionLine, 2))
+                    
+                    return
+                }
+                
+            }
+            
+            NSBeep()
```
Comment (markohlebar) [inline ImportExtension/AddImportOperation.swift]: Interesting approach... Does `NSAlert` work?

File: ImportExtension/AddImportOperation.swift [hunk 2]
```diff
@@ -115,7 +145,27 @@ fileprivate extension XCSourceTextBuffer {
         
         let importBufferArray = self.lines.subarray(with: NSMakeRange(0, atLine)) as NSArray as! [String]
         
-        return importBufferArray.contains(importString) == false
+        var sanitizedImportString: String = importString
+
+//        if let position = importString.characters.index(of: "/") {
```
Comment (markohlebar) [inline ImportExtension/AddImportOperation.swift]: Remove commented lines pls if we don't need them.

Comment (clementpadovani) [reply ImportExtension/AddImportOperation.swift]: Whoops, fixed.

File: ImportExtension/AddImportOperation.swift (hunk 1 above)
Comment (clementpadovani) [reply ImportExtension/AddImportOperation.swift]: Hey good call!

untitled 2

I’d have to rewrite that to make use of the alert.

**Comment by clementpadovani** [comment]
@markohlebar after some fiddling I’m getting some fixed results with an alert. the system doesn’t seem to really like having an alert come from an extension. And TBH it doesn’t really feel right

**Comment by markohlebar** [comment]
@ClementPadovani what happened? I just tried it and it worked fine. Did you do 

```
DispatchQueue.main.async {
            //present NSAlert
        }
```

? 
The [docs](https://developer.apple.com/reference/xcodekit/xcsourceeditorcommand/2097278-performcommandwithinvocation?language=objc) say `There are no guarantees about the thread or queue on which the cancellation handler is invoked.`. Is that the problem?

**Comment by markohlebar** [comment]
@ClementPadovani re-read that last thing again, not sure it's relevant but it might be that there are no guarantees on which thread the `performCommandWithInvocation` is called either.

**Comment by clementpadovani** [comment]
@markohlebar Yes I had this issue, however instead of wrapping it around a `DispatchQueue` I used `OperationQueue.mainQueue`.

**Comment by markohlebar** [comment]
@ClementPadovani what's the actual problem, is it flaky in presentation even when you dispatch it on the main queue?

**Comment by clementpadovani** [comment]
@markohlebar Okay so I figured out the issue. The alert was actually being displayed, however it was hidden behind Xcode. https://github.com/markohlebar/Import/pull/11/commits/816db1d71fb32e775f156c28f5440aad829c95b9 fixes that issue.

I had to pass the completion handler as a parameter since calling it would kill the extension. Thus preventing the alert being shown and doing its work.

**Comment by markohlebar** [comment]
Hey @ClementPadovani checked the PR, looks good! 

However, I'd say that there is kind of a bias as to what "Remove Import" would do. At the first hunch it would remove the import from the top, so that's kinda bad. 

On the other hand, I would expect "Cancel" to just remove the import statement as the "Remove Import" button does at the moment. There is no use for it anyways, so as a flow of importing something I think removing it as a default action would be OK, what do you think?

**Comment by clementpadovani** [comment]
@markohlebar fixed with d8c60d7

**Comment by markohlebar** [comment]
@ClementPadovani awesome!

**Pull request merged**